### PR TITLE
Remove an unused parameter

### LINKED ISSUE #13943: Parameter 'propq' in CMS_ReceiptRequest_create0_ex() is documented but unused
In `CMS_ReceiptRequest_create0_ex()`, parameter `propq` is not used at all. However, in the corresponding [document](https://github.com/openssl/openssl/blob/master/doc/man3/CMS_get1_ReceiptRequest.pod), it says

> CMS_ReceiptRequest_create0_ex() creates a signed receipt request structure... **the property query propq are used when retrieving algorithms from providers.**

So I guess either it's necessary to add "`propq` is ignored " to document or the implementation of `CMS_ReceiptRequest_create0_ex()` may have potential bugs.

### PR DESCRIPTION
**Pull request by richsalz**
In reply to https://github.com/openssl/openssl/pull/14064#issuecomment-823118200

All that changes is documenting the one, existing, parameter.

 Fixes #13943

### CODE REVIEW

File: doc/man3/CMS_get1_ReceiptRequest.pod [hunk 1]
```diff
@@ -33,7 +33,7 @@ If I<receiptList> is NULL the allOrFirstTier option in I<receiptsFrom> is used
 and set to the value of the I<allorfirst> parameter. If I<receiptList> is not
 NULL the I<receiptList> option in I<receiptsFrom> is used. The I<receiptsTo>
 parameter specifies the I<receiptsTo> field value. The library context I<libctx>
-and the property query I<propq> are used when retrieving algorithms from providers.
+is used interanlly to find the public random generator.
```
Comment (FdaSilvaYY) [inline doc/man3/CMS_get1_ReceiptRequest.pod:36]: typo: interanlly

Comment (richsalz) [reply doc/man3/CMS_get1_ReceiptRequest.pod]: sigh.  fixed, thanks!

**Comment by t8m** [comment]
No, OTC wants the parameter removed as it is unnecessary and confusing. This is new API in 3.0 so we are free to remove it.

**Comment by richsalz** [comment]
> No, OTC wants the parameter removed as it is unnecessary and confusing. 

Ah, I misread.  Updated commit pushed.

**Comment by mspncp** [comment]
Your commit message does not match the updated commit anymore.

**Comment by richsalz** [comment]
sorry, jugggling too many things at once.  fixed the commit.

**Comment by slontis** [comment]
Please add Fixes #13943 to the commit.

**Comment by t8m** [comment]
> Please add Fixes #13943 to the commit.

Or we can add that during merging.

**Review by t8m — APPROVED** [review]

**Review by mspncp — APPROVED** [review]

**Comment by t8m** [comment]
Merged to master. Thank you for the contribution!

**Pull request closed without merging**